### DISCUSSION TITLE
Wormhole WETH sent from Solana to WETH on Ethereum - Missing

### DISCUSSION THREAD

**Discussion by jacksparrow9201** [wormhole-foundation/wormhole#331] [Problems / bugs] [answered]
First i've sent 0.9 WETH from ETH to SOLANA which worked, i could not use it on any of SOLANAS exchanges so i decided to send it back to metamask using wormhole again, funds never arrived after 10 hours? please help

**Comment by leoluk** [comment]
Please provide transaction hashes

**Reply by jacksparrow9201** [reply]
Hi, here is my failed transaction from the phantom wallet

https://explorer.solana.com/tx/uHEgoj2BrXtrhKsMmZK1yWfrd1dzowpt9fUBf4fMjys2sp444fStpLYS9ByinCCezQrAVFA8SmV4P1DWqSuxwip

**Comment by leoluk** [comment, marked as answer]
It looks like that's the wrong transaction, here's the correct one: https://explorer.solana.com/tx/8EDeFXU6nw1HFrcwPzV9SBMsKvaKCAMyc65WPfvTDN4pRM4foGD9G4XapAVVKuLDRiKc1k3c1MKqgNjFyyX4pcd

... which was successfully processed by Wormhole.

wormholebridge.com should show a list of pending transactions and let you retry the Eth submission. If this isn't the case, you can [create an issue with the authors of wormholebridge.com](https://github.com/solana-labs/oyster/issues) or manually submit the signed VAA bytes below to the submitVAA method of the Eth contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

---

Digest: 48b54f7fffec9973373c83bcdf7acc86860845488df272a8bfb7edb8d8940615

Signed VAA bytes:

> 01000000010d000e924591e0e97447fb2409626772360e795939ffabfd2088ecdfe503fc699d2376823de73708c854237755b71de81701fdd5bec058001a0bbbf242c57de1d6f300014aaf50eca8d9185b220c93cb38164cd46a1d47d7fac897939488e0602c86876b2919fd816bb00f04b55788b88c40d59cbe7c60d8ad316417d607ec8ee8c00344000200dba63dee04354789dd4428be22e22360c51c33b07ce96f7b474e65e0a09f1464f3e7a6f8ce0ffa72545a1b925b6d433dbfa3a373e99cf88587a44925b4315d0003207394f4726a75d6da426db4b3475e7edd17ecc96c8dc0b9abce4213fda97c7875ef0b2e64c0e8ef490e7d7759388eefe37cbbd2aef0242e99e5f0dc6ba8c2bc0005218bb897e2ff37d6405b6ff45a983828fba2dfdfbb6dd6b1682511b87b1fa5d65cbbb753cfc2c119a7d892e2f1521429fedae4f35214804dca17ae465b5c3f02000600280891330f7c526481854d2290c5873f17b0d81f5a2bca66c9b1e198f726e3676cf43c53ce67f31c6ef3733c9e1f42205d04c0854352e646df466737863b670107d122cb8bfcdc039ca2f59e99b8618c08c394494a8a4611677a03463ca510e161020a3e4c0db65b348e139b5c4e4df3dcf867e45a82a2ff027b10759d1660fdc0000813f119c6c2eddfc4c38d1dddd1bd6ed2c235b93a017b8b504044c3ec16d4584421d2e541374c744bf434fcebd94f89582cebeb6452c37b054023b7d25b1b46e9010a5b93c6b0a81d30cc0ce5ea06f742fef9a302946942035bdbd2034c0bd1206b802db77520f67625660095e55480f08c27a4ffb093feee25ca4900dedc41cf98e3010dac7c7d7fba833a047b23b2c47b8ddc621e9d85157287ce6ae63b480ff6205c7f306036aa60fd76e2fb23162e48035feaecbe289a28147c5f8ee28424e6ae601d01107eda7a10b95014912651dfd9dcecb2b32b6a7434c17eb69e77b650398c5104cc5b1f31cfa1828afad6bbdeeef2510acbd60fcc9b3f2c5af0073bab57dff017470011f37ad740f7667438d9fd4e101a684f5cc28db60f93da3d293e974ae5d2e513804a516f43f23e0afb775b40e28d9dae062105395dd59dde74576f39df30f0dc4701125138b623043881a6de0d9c88f19a9c68cde7aeffae291c5305deb095830ac3144c9b6c2e21111af01440d9df06ffad2a9ef96667f7e9bd58dd321d812362228e0161202df91000012d4a01029c85f35472e559b635275b162a2842a4f6a69df6b56f755dc07bbbe7130714ea000000000000000000000000fc48bdf7e2245c6f86aa0c5289df96b0848c029602000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2090000000000000000000000000000000000000000000000000000000035a4e900

**Reply by jacksparrow9201** [reply]
I have 0 WETH in my phantom wallet and 0 WETH in my metamask, my funds are missing after wormhole from SOLANA to metamask ETH chain , how do i retrieve my 0.9 WETH? ive been using SOLANA for a day and I am lost, thanks for understaning

**Reply by leoluk** [reply]
Please follow the instructions above. The UI you're using didn't properly submit the finalized transaction on ETH and you need to retry it.

**Reply by jacksparrow9201** [reply]
Thanks a lot i got it now in my metamask

**Reply by d0llarbill** [reply]
Hello,

I'm having this same issue with bridging TUSD from SOL to ETH using the wormhole connected to phantom wallet on SOL and metamask on ETH mainnet. I did not have enough for gas on metamask to complete the bridge and closed the gas page. After this ocurred I can't find the transaction in SOL under recent transactions. Something is trying to load and won't appear. Are you able to provide some additional clarification regarding how to manually submit the signed VAA bytes mentioned above. I'm trying to figure out how to obtain this information in order to resolve the issue based on your instructions. Phantom wallet transaction: https://explorer.solana.com/tx/2rku2ffeS97SWfETYy2WrMDfCua9o4fomrAuvN7DAeE3seiVHfEYYvU8wQhFyfbVh5V94KccKuxV8aL6xoV8wLm3 

Thanks in advance

**Reply by leoluk** [reply]
@d0llarbill please create a new issue at https://github.com/solana-labs/oyster/issues rather than hijacking an existing thread
